testData/solutions/win32.json5

```diff
@@ -2742,5 +2742,117 @@
         "start": [],
         "stop": [],
         "isInstalled": [] // always reported as not installed. Should be changed once GPII-1998 is fixed
+    },
+
+    "net.gpii.browserExtension": {
+        "name": "UIO+ Browser Extension",
+        "contexts": {
+            "OS": [{
+                    "id": "win32",
+                    "version": ">=5.0"
+            }]
+        },
+        "settingsHandlers": {
+            "chromeconf": {
+                "type": "gpii.settingsHandlers.webSockets",
+                "options": {
+                    "path": "net.gpii.browserExtension"
+                },
+                "capabilities": [
+                    "http://registry\\.gpii\\.net/common/lineSpace",
```

Comment:
I think it's perfectly fine to have this entry (and those below) - but just an FYI that the system can (and will) infer most of these terms from the capabilitiesTransformations block below.. but leave them in - I think it's nice to be explicit about the supported terms

Reply:
Or equally likely, that the system doesn't extract the paths from nested transforms particularly well :)